docs/features/producer_granule_id_duplication.md

```diff
@@ -0,0 +1,222 @@
+---
```

Comment:
Should we name this file something like `granule_uniquification.md` instead of `producer_granule_id_duplication`? Just looking to make it easier to find in the future when I forget this ever happened.

Reply:
Excellent point - [45bd997](https://github.com/nasa/cumulus/pull/4048/commits/45bd997a4af59ed025c613d7947e6e8f8bf529e0)